README.md

```diff
@@ -9,3 +9,14 @@ This runs locally watching for changes and live reloading.
 ```
 make run
 ```
+
+Open http://localhost:4000 in your browser.
+
+## To run locally with local crossplane docs
+Ensure `$(GOPATH)/src/github.com/crossplaneio/crossplane/docs` is present.
+
+```
+make run_docs_local
```

Comment:
This requires node to be installed. It'd be good to have a small requirements section I believe.

Reply:
I had to brew install npm, will add that to the instructions

Reply:
brew instruction assumes that the OS is Mac OS X though